Main/Source/ExtendedWPFToolkitSolution/Src/Xceed.Wpf.Toolkit/Primitives/UpDownBase.cs

```diff
@@ -453,7 +453,6 @@ internal UpDownBase()
       this.AddHandler( Mouse.PreviewMouseDownOutsideCapturedElementEvent, new RoutedEventHandler( this.HandleClickOutsideOfControlWithMouseCapture ),
 
 true );
-      this.IsKeyboardFocusWithinChanged += this.UpDownBase_IsKeyboardFocusWithinChanged;

```

Comment:
As discussed f2f, i will merge this change, since it fixes the undo issue. We need to be aware that this may have sideeffects  in the DerivedClasses, though (e.g. pressing return for NumericUpDown). If there are any issues coming up during testing we might need to revert this fix.